the-content-guide/readme.md

```diff
@@ -1,20 +1,129 @@
+
+**Bold** and *italic* words make sure that highlighted concepts are cleared up. *Short paragraphs* are there to make the text understandable for the reader. *Clear quotes*, then, invite the reader to keep on reading.
+=======
+The *intro* is usually the first paragraph, and also supports a well chosen title. How well an introduction activates the reader determines whether she or he will continue reading the article, this often happens within seconds of seeing the intro for the first time. The intro can consist of a promise that you make, they were lured in with the title as a hook and now they want to know if the content is worth there precious time.
+
+**Bold** en *italic* words ensure that highlighted concepts are cleared up. *Short paragraphs* are there to make the text understandable for the reader. *Clear quotes*, then, invite the reader to keep on reading.
+>>>>>>> origin/improving-the-content-duide
+
+## Visitor behavior
+
+With the ever-growing content, the bad and the good, content developers should know how readers behave on the web, and how they can use that knowledge towards their own advantage.
+
+According to Jacob Nielsen of the [Nielsen Norman Group](http://www.nngroup.com/articles/how-long-do-users-stay-on-web-pages/), a visitor spends an average of less than a minute on a webpage. The visitor usually only has time to read a quarter of the text presented and if the content has a clear message or benefit, the chances it will be read are heightened. Quality content aims to give the reader new insights about a specific topic.
+
+## Search Engine Optimisation (SEO) or keyword research, here we go:
+
+Search engine optimisation is a way to (sort of) ensure that people find your website through search engines like Google, Bing, Google and Google.
+
+We believe that the quality, originality and creative components of created content always stands above SEO strategy. However, we do acknowledge SEO as an important strategy when it comes to the visibility of the content. It would be a shame for a written masterpiece to float on the web and not be picked up by an audience.
+
+SEO can be done in many ways, and has mostly to do with figuring out what words your target audience searches for and then using those words throughout your article. At New Atoms we don't really do SEO, but it is good to be aware of it as a writer.
+
+### Tackle marketing priorities for SEO
+
+One goal of content marketing is that businesses want to increase traffic. Top named challenges and priorities for marketing professionals–besides creating more _high quality content_ in lesser time—are:
```

Comment:
'One goal of content marketing is increased traffic for businesses.'

'Top-named'

change dashes to parentheses: '(besides creating more high-quality content_in lesser time) are:'